### PR TITLE
Jjdzr4 mw 37 menu user logged

### PR DESCRIPTION
**Pull request by lyingparachute**
LOMBOCK mostly

### CODE REVIEW

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/DishData.java [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+
 import javax.persistence.*;
 import javax.validation.constraints.NotEmpty;
 import java.util.ArrayList;
 import java.util.List;
 
 @Entity
-public class DishData {
+public @Data
+class DishData {
```
Comment (mateuszostrowski) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/DishData.java:13]: shoudn't it be like <br /> `@Data` <br /> `public class DishData` ?

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/DishData.java [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
     private static final String NOT_EMPTY_MESSAGE = "Pole nie może być puste";
 
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @Getter
```
Comment (mateuszostrowski) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/DishData.java:18]: Do we need this `@Getter` when we used `@Data` already?

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/ProductData.java [hunk 3]
```diff
@@ -1,17 +1,22 @@
...
 @Entity
+@Data
 public class ProductData {
 
     private static final String NOT_EMPTY_MESSAGE = "Pole nie może być puste";
     private static final String POSITIVE_OR_ZERO_MESSAGE = "Wartość nie może być ujemna";
 
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @Getter
```
Comment (mateuszostrowski) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/ProductData.java:19]: Do we need this `@Getter` when we used `@Data` already?

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/UserData.java [hunk 4]
```diff
@@ -1,11 +1,16 @@
...
+import lombok.Getter;
+
 import javax.persistence.*;
 
 @Entity
+@Data
 public class UserData {
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @Getter
```
Comment (mateuszostrowski) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/UserData.java:13]: Do we need this `@Getter` when we used `@Data` already?

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/UserLogin.java [hunk 5]
```diff
@@ -1,11 +1,16 @@
...
+
 import javax.persistence.*;
 
 
 @Entity
+@Data
 public class UserLogin {
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @Getter
```
Comment (mateuszostrowski) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/UserLogin.java:13]: Do we need this `@Getter` when we used `@Data` already?

File: myFit-webApp/src/main/resources/userData.json [hunk 6]
```diff
@@ -5,11 +5,11 @@
     "password": "123"
```
Comment (mateuszostrowski) [inline myFit-webApp/src/main/resources/userData.json:5]: shouldn't we add this file to .gitignore?

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/DishData.java (hunk 2 above)
Comment (lyingparachute) [reply myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/DishData.java:18]: We don't want to have setter on our ID

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/ProductData.java (hunk 3 above)
Comment (lyingparachute) [reply myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/ProductData.java:19]: Same as above

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/UserData.java (hunk 4 above)
Comment (lyingparachute) [reply myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/UserData.java:13]: Same as above

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/UserLogin.java (hunk 5 above)
Comment (lyingparachute) [reply myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/UserLogin.java:13]: Same as above

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/DishData.java (hunk 1 above)
Comment (lyingparachute) [reply myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/DishData.java:13]: It doesn't matter, works both ways

**Review by mateuszostrowski — APPROVED** [review]

File: myFit-webApp/src/main/resources/templates/fragments/anonymous-user-index.html [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+<head>
+    <title th:text="#{user.title}"></title>
+</head>
+<body sec:authorize="isAnonymous()">
+<div>
+    <h1>Jestem niezalogowany?</h1>
+    <div class="p-5 text-center bg-light">
+        <h1 class="mb-3">Welcome to MyFitApp</h1>
+        <div class="description ">
+            <h1>    Hello ,Welcome To My official Website
```
Comment (miosz) [inline myFit-webApp/src/main/resources/templates/fragments/anonymous-user-index.html:13]: is this template for temporary usage only?

File: myFit-webApp/src/main/resources/templates/fragments/authenticated-user-index.html [hunk 8]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="https://www.thymeleaf.org" xmlns:sec="https://www.thymeleaf.org/thymeleaf-extras-springsecurity3">
+<html lang="en">
+<head>
+    <title th:text="#{user.title}"></title>
+</head>
+<body sec:authorize="isAuthenticated()">
+<div>
+    <h1 th:inline="text">Hello [[${#httpServletRequest.remoteUser}]]!</h1>
+    <h1>Czy to działa?</h1>
```
Comment (miosz) [inline myFit-webApp/src/main/resources/templates/fragments/authenticated-user-index.html:10]: do want this message here?

**Review by miosz — APPROVED** [review]

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/SecurityConfig.java [hunk 9]
```diff
@@ -29,9 +29,7 @@ protected void configure(final HttpSecurity http) throws Exception {
                 .permitAll()
                 .and()
                 .logout().logoutSuccessUrl("/?logout")
-                .permitAll()
-                        .and().csrf().disable();
```
[the remark this answers is not in the document]
Comment (lyingparachute) [reply myFit-webApp/src/main/java/com/infoshare/myfitwebapp/SecurityConfig.java:33]: Nope

File: myFit-webApp/src/main/resources/templates/fragments/anonymous-user-index.html (hunk 7 above)
Comment (lyingparachute) [reply myFit-webApp/src/main/resources/templates/fragments/anonymous-user-index.html]: It will be used later on

**Pull request merged**